scripts/manage_recordings.py

```diff
@@ -0,0 +1,141 @@
+#   azure-keyvault-keys recordings to the assets repo with:
+#
+#     `python scripts/manage_recordings.py push sdk/keyvault/azure-keyvault-keys/assets.json`
+#
+# - In addition to "push", you can also use the "restore" or "reset" verbs in the same command format.
+#
+# For more information about how recording asset synchronization, please refer to
+# https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/documentation/asset-sync/README.md.
+
+ # Load environment variables from user's .env file
+
+CONTAINER_NAME = "azsdkengsys.azurecr.io/engsys/testproxy-lin"
+GIT_TOKEN = os.getenv("GIT_TOKEN", "")
+GIT_OWNER = os.getenv("GIT_COMMIT_OWNER", "")
+GIT_EMAIL = os.getenv("GIT_COMMIT_EMAIL", "")
+
+
+# ----- HELPERS ----- #
+
+
+discovered_roots = []
+
+def ascend_to_root(start_dir_or_file: str) -> str:
+    """
+    Given a path, ascend until encountering a folder with a `.git` folder present within it. Return that directory.
+
+    :param str start_dir_or_file: The starting directory or file. Either is acceptable.
+    """
+    if os.path.isfile(start_dir_or_file):
+        current_dir = os.path.dirname(start_dir_or_file)
+    else:
+        current_dir = start_dir_or_file
+
+    while current_dir is not None and not (os.path.dirname(current_dir) == current_dir):
+        possible_root = os.path.join(current_dir, ".git")
+
+        # we need the git check to prevent ascending out of the repo
+        if os.path.exists(possible_root):
+            if current_dir not in discovered_roots:
+                discovered_roots.append(current_dir)
+            return current_dir
+        else:
+            current_dir = os.path.dirname(current_dir)
+
+    raise Exception(f'Requested target "{start_dir_or_file}" does not exist within a git repo.')
+
+
+def delete_container() -> None:
+    """Delete container if it remained"""
+    proc = subprocess.Popen(
+        shlex.split(f"docker rm -f {CONTAINER_NAME}"),
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        stdin=subprocess.DEVNULL,
+    )
+    output, stderr = proc.communicate(timeout=10)
+    return None
+
+
+def get_image_tag(repo_root: str) -> str:
+    """Gets the test proxy Docker image tag from the target_version.txt file in /eng/common/testproxy"""
+    version_file_location = os.path.relpath("eng/common/testproxy/target_version.txt")
+    version_file_location_from_root = os.path.abspath(os.path.join(repo_root, version_file_location))
+
+    with open(version_file_location_from_root, "r") as f:
+        image_tag = f.read().strip()
+
+    return image_tag
+
+
+# ----- CORE LOGIC ----- #
+
+
+if not (GIT_TOKEN and GIT_OWNER and GIT_EMAIL):
+    raise ValueError(
+            "GIT_TOKEN, GIT_COMMIT_OWNER, and GIT_COMMIT_EMAIL environment variables must be set, "
+            "either in-process or in a .env file"
+        )
+
+# Prepare command arguments
+try:
+    command_verb = sys.argv[1].lower()  # "restore", "reset", or "push"
```

Comment:
I won't insist given that you're just using these two values directly, but `argparse` makes this fairly straightforward with help text etc.